doc/UserGuide.md

```diff
@@ -35,19 +35,21 @@ Format: `help`
  
 ### Adding a person: `add`
 Adds a person to the address book<br>
-Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...` 
+Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
  
 > Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
 > items with `...` after them can have multiple instances. Order of parameters are fixed. 
 > 
+> Address Format: `BLOCK, STREET, UNIT, POSTAL_CODE`
```

Comment:
Good to see that you updated the user guide, but you can consider combining the address format directly with the full `add` command format :)